dev/snippets/nav/bar_basic.cljs

```diff
@@ -1,11 +1,14 @@
 #_
 (:require [om-bootstrap.button :as b]
-          [om-bootstrap.nav :as n])
+          [om-bootstrap.nav :as n]
+          [om-tools.dom :as d :include-macros true])
 
 (n/navbar
- {}
+ {:brand (d/a {:href "#"}
+              "Navbar")
+  :toggle-nav-key 0}
  (n/nav
-  {:collapsible? true}
+  {:key 0}
```

Comment:
The way the code is currently setup, you have to add a :toggle-nav-key or a :toggle-button to the navbar props in order for the header to render with a toggle-button:
```
(defn render-header [owner bs]
  (d/div {:class "navbar-header"}
         (if (u/strict-valid-component? (:brand bs))
           (u/clone-with-props (:brand bs) {:class "navbar-brand"})
           (d/span {:class "navbar-brand"} (:brand bs)))
         (when (or (:toggle-button bs)
                   (:toggle-nav-key bs))
           (render-toggle-button owner bs))))
```

Also, the :collapsible? property that we feed into `nav` requires that a :toggle-nav-key be defined, and whichever navbar child has that key will be assigned :collapsible? true
```
                  collapsible? (when (:toggle-nav-key bs)
                                 (= (:key opts) (:toggle-nav-key bs)))
```